src/app/ExportPanel.tsx

```diff
@@ -25,6 +25,17 @@ export const ExportPanel = observer(() => {
     onExportMenuItemClick();
   };
 
+  const onExportAsURL = () => {
+    console.log(app.diagram.fields.length);
+    const encodedJsonDiagram = window.btoa(app.diagram.toJson());
+    const protocol = window.location.protocol;
+    const domain = window.location.hostname;
+    const port = window.location.port;
+    const urlWithJson = `${protocol}//${domain}:${port}?Diagram=${encodedJsonDiagram}`;
+    navigator.clipboard.writeText(urlWithJson);
```

Comment:
Also set the current page url to contain the query parameters. Tips: `history.pushState` API